commit 950ed6caba2cbdccb69451d416d6916b1a1bcbeb (HEAD -> refs/heads/master, refs/remotes/github/master)
Author: Robb Matzke <matzke1@llnl.gov>
Date:   Wed Oct 31 11:52:57 2018 -0400

    $CC needs to be saved in tup.config

diff --git a/build.sh b/build.sh
index fa00c4e..9d81b28 100755
--- a/build.sh
+++ b/build.sh
@@ -18,6 +18,7 @@ else
 fi
 LDFLAGS="$LDFLAGS -lm"
 : ${CC:=gcc}
+echo "CONFIG_CC=$CC" >tup.config
 case "$os" in
 	Linux)
 	plat_files="$plat_files ../src/compat/dummy.c"
