diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2f3b6b3..4c4da41 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -531,13 +531,13 @@ ENDIF (DOXYGEN_FOUND)
 
 # we enable rpath support for APPLE, this probably goes against policy
 # linux distributions?
-IF(APPLE)
   SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
   SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
   LIST(FIND CMAKE_PLATFORM_IMPLICIT_LINK_DIRECTORIES "${CMAKE_INSTALL_PREFIX}/lib" isSystemDir)
   IF("${isSystemDir}" STREQUAL "-1")
     SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
   ENDIF("${isSystemDir}" STREQUAL "-1")
+IF(APPLE)
   SET(CMAKE_MACOSX_RPATH TRUE)
 ENDIF(APPLE)
 
